settings.gradle.kts

```
@@ -25,3 +26,4 @@ project(":rest-tmi").name = "twitch4j-messaginginterface"
 project(":pubsub").name = "twitch4j-pubsub"
 project(":graphql").name = "twitch4j-graphql"
 project(":twitch4j").name = "twitch4j"
+project(":twitch4j-ktx").name = "twitch4j-ktx"
```

should we change the module name to `twitch4j-kotlin`? Most projects i've seen so far use the `-kotlin` suffix for their kotlin artifacts on mvncentral

---

I am very used to Android haha, so I might be biased to KTX as a lot of libraries are suffixed with that.
My thought: As this library only really contains Kotlin Extension Functions and not actually functionality, KTX was a more descriptive term for it.
I am totally fine with naming it twitch4j-kotlin though. 😋

---

```suggestion
project(":kotlin").name = "twitch4j-kotlin"
```

and rename directory to `kotlin`